repository/n.json

```diff
@@ -1132,6 +1132,18 @@
 				}
 			]
 		},
+		{
+			"name": "NpmRunner",
+			"details": "https://github.com/kennethnym/sublime-npm-runner",
+			"author": "kennethnym",
```

Comment:
If the author is just your GitHub user name you don't need to provide it. Package Control will take that info from the repo.

Reply:
Got it, I'll change it

Reply:
👋🏻